package co.com.siigo.certificacion.moduloprueba.exceptions;

public class Exception {
    private Exception(){}
}